openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -66,50 +139,70 @@ objects:
     metadata:
       name: ${NAME}
       labels:
-        name: ${NAME}
-        app: ${APP_GROUP}
+        app: ${APP_NAME}${SUFFIX}
+        name: ${NAME}${SUFFIX}
         app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
     stringData:
       seed: ${WALLET_SEED}
       key: ${WALLET_KEY}
     type: Opaque
+  
+  - kind: Secret
+    apiVersion: v1
+    metadata:
+      name: ${NAME}-api${SUFFIX}
+      labels:
+        name: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
+    stringData:
+      admin-api-key: ${ADMIN_API_KEY}
+    type: Opaque
 
   - kind: DeploymentConfig
     apiVersion: v1
     metadata:
       name: ${NAME}
       labels:
-        name: ${NAME}
-        app: ${APP_GROUP}
+        app: ${APP_NAME}${SUFFIX}
+        name: ${NAME}${SUFFIX}
         app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
     spec:
       replicas: 1
       selector:
         name: ${NAME}
       strategy:
-        type: Rolling
+        type: Recreate
```

Comment:
Why the change in the deployment strategy.  I don't think we want to use a  `Recreate` strategy on agents, that would cause an unnecessary interruption in service.

Reply:
@esune suggested that correct? or was that just for builds

Reply:
changed back to rolling